config/manifests/kustomization.yaml

```diff
@@ -14,7 +14,7 @@ patches:
   target:
     group: operators.coreos.com
     kind: ClusterServiceVersion
-    name: opendatahub-operator.v0.0.0
+    name: rhods-operator.v2.0.0
```

Comment:
should be v2.1.0 ? same as the one set in Makefile

Reply:
Updated